arviz/tests/helpers.py

```diff
@@ -14,6 +14,10 @@
 import scipy.optimize as op
 import torch
 
+import logging
+
+_log = logging.getLogger(__name__)
```

Comment:
should we use `logging.getLogger('arviz')` instead of `__name__`?

Reply:
I think it's functionally equivalent, but this comment swayed me enough to the "explicit over implicit".

Will make the change

Reply:
Ohh, I confused this with the `__init__.py` at the root directory of arviz.

I think this is better because itll log under `arviz.tests.helpers` which will capture where the logs are coming from. If we use just `arviz` it becomes harder to pinpint. 

The logging functionality is exactly the same because python logger inherit from the root logger, which is configured in '__init__.py` in arviz's root directory

My recommendation is to keep it as such, absent of some other knowledge that Im missing.